doc/src/manual/environment-variables.md

```diff
@@ -74,6 +79,10 @@ Julia tries to find a project directory that contains `Project.toml` or
 `JuliaProject.toml` file from the current directory and its parents.  See also
 the chapter on [Code Loading](@ref).
 
+!!! note
+
+    See note at top as `JULIA_PROJECT` cannot be set in `~/.julia/config/startup.jl`
```

Comment:
Instead of linking to the top, perhaps just include a shorter explanation here? Maybe something like:
```
`JULIA_PROJECT` must be defined before starting julia; defining it in `startup.jl` is too late in the startup process.
```